server/functions/framework/compiled_function.go

```diff
@@ -714,6 +745,10 @@ func (c *CompiledFunction) analyzeParameters() (originalTypes []*pgtypes.Doltgre
 			if err != nil {
 				return nil, err
 			}
+			// text type need be interpreted as unknown to be resolved into type that a function fits with.
```

Comment:
This seems like it might be horribly wrong. What breaks if this is removed? Function resolution isn't complete yet, it's just as completed as we needed it (at the time). Fixing the invalid `unknown` assumptions last time wasn't fun.

Reply:
We can remove this, at the very least all tests pass if it's deleted.

Reply:
Done